woocommerce/woocommerce-android
# [Migrate Simple Payments] Tracking

Closes: #11203


### Description

The PR adds tracking of the events:
* `simple_payments_migration_sheet_add_custom_amount`
* `simple_payments_migration_sheet_shown`

### Testing instructions

Use logcat with "track" filter
* More -> Payments -> Collect Payments -> Add A custom amount 
* Notice events tracked mentioned above
